manifests/extension.pp

```diff
@@ -179,24 +179,30 @@
     require => $package_depends,
   }
 
-  # Ubuntu/Debian systems use the mods-available folder. We need to enable
-  # settings files ourselves with php5enmod command.
+  # Ubuntu/Debian systems use the mods-available folder. We need to handle
+  # settings files ourselves with CLI commands.
   $ext_tool_enable   = pick_default($::php::ext_tool_enable, $::php::params::ext_tool_enable)
+  $ext_tool_disable  = pick_default($::php::ext_tool_disable, $::php::params::ext_tool_disable)
   $ext_tool_query    = pick_default($::php::ext_tool_query, $::php::params::ext_tool_query)
   $ext_tool_enabled  = pick_default($::php::ext_tool_enabled, $::php::params::ext_tool_enabled)
+  $final_sapi = $sapi ? {
+    'ALL'   => 'cli',
+    default => $sapi,
+  }
 
   if $::osfamily == 'Debian' and $ext_tool_enabled {
-    $cmd = "${ext_tool_enable} -s ${sapi} ${lowercase_title}"
+    if $ensure == 'absent' {
+      $cmd = "${ext_tool_disable} -s ${final_sapi} ${lowercase_title}"
```

Comment:
nope, `final_sapi` should still be used with `phpquery` (in the `unless` or `onlyif` statements) as phpquery doesnt support the `ALL` option.

```
usage: phpenmod [ -v ALL|php_version ] [ -s ALL|sapi_name ] module_name [ module_name_2 ]
```

```
usage: phpdismod [ -v ALL|php_version ] [ -s ALL|sapi_name ] module_name [ module_name_2 ]
```

```
usage: phpquery [ -d ] [ -q ] -v version_name -s sapi_name [ -m module_name ] [ -M ] [ -S ] [ -V ]
```